pkg/imgpkg/v1/describe.go

```diff
@@ -98,7 +99,7 @@ func DescribeWithRegistryAndSignatureFetcher(bundleImage string, opts DescribeOp
 		return Description{}, fmt.Errorf("Only bundles can be described, and %s is not a bundle", bundleImage)
 	}
 
-	allBundles, err := newBundle.FetchAllImagesRefs(opts.Concurrency, opts.Logger, sigFetcher)
+	allBundles, err := newBundle.FetchAllImagesRefs(opts.Concurrency, opts.MaxDepth, opts.Logger, sigFetcher)
```

Comment:
Before we call `FetchAllImagesRefs` do you mind checking that the depth is >= 0? Just to ensure people do not try to get a negative number in